packages/react-components/src/Dropdown.tsx

```diff
@@ -108,25 +107,36 @@ function BaseDropdown<Option> ({ allowAdd = false, children, className = '', def
     />
   );
 
-  return isButton
-    ? <SUIButton.Group>{dropdown}{children}</SUIButton.Group>
-    : (
-      <Labelled
-        className={classes('ui--Dropdown', className)}
-        help={help}
-        isFull={isFull}
-        label={label}
-        labelExtra={labelExtra}
-        withEllipsis={withEllipsis}
-        withLabel={withLabel}
-      >
-        {dropdown}
-        {children}
-      </Labelled>
-    );
+  return (
+    <div className={className}>
+      {isButton
+        ? <SUIButton.Group>{dropdown}{children}</SUIButton.Group>
+        : <Labelled
+          className='ui--Dropdown'
+          help={help}
+          isFull={isFull}
+          label={label}
+          labelExtra={labelExtra}
+          withEllipsis={withEllipsis}
+          withLabel={withLabel}
+        >
+          {dropdown}
+          {children}
+        </Labelled>
+      }
+    </div>
+  );
 }
 
 const Dropdown = React.memo(styled(BaseDropdown)`
+  .ui.selection.dropdown {
+    font-size: 1rem;
+    color: #000000;
+    font-weight: 400;
+    border: 1px solid #DFDFDF;
+    border-top-left-radius: 0;
+    border-bottom-left-radius: 0;
+  }
```

Comment:
Sadly this does break and conflict with what is in `styles/semantic` - prefer keeping it in there explicitly (with the theme overrides)

And additional comment, rather never use pure black, #4e4e4e is the base color (per theme) for light.